src/notation/internal/notationinteraction.cpp

```diff
@@ -2771,6 +2771,41 @@ void NotationInteraction::changeEditElement(EngravingItem* newElement)
     }
 }
 
+bool NotationInteraction::isEditAllowed(QKeyEvent* event)
+{
+    if (!m_editData.element) {
+        return false;
+    }
+
+    m_editData.modifiers = event->modifiers();
+    m_editData.key = event->key();
+
+    if (m_editData.element->isEditAllowed(m_editData)) {
+        return true;
+    }
+
+    if (event->modifiers() & Qt::KeyboardModifier::AltModifier) {
+        return false;
+    }
+
+    if (m_editData.element->isTextBase()) {
+        return false;
```

Comment:
This code looks strange. Is it a typo?

Reply:
No, why strange?

Reply:
Looks like we can't edit text elements anymore

Reply:
Yes, we should get the answer about the ability to edit the text above:
```
 if (m_editData.element->isEditAllowed(m_editData)) {
        return true;
    }
```